IrConsoleApp/Lecture6/Course.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using IrynaZharnasek.Lecture6;
+
+namespace IrynaZharnasek.Lecture6
+{
+    public class Course
+    {
+        private string course;
+
+        private Student[] arrayOfStudents = new Student[100];
+        private Mentor[] arrayOfMentors = new Mentor[100];
```

Comment:
As far as I understand, the current logic is the following: one course can have 100 Mentors. But I would say it's better the logic be the following: one course can have one Mentor. But Courses Library can have 100 mentors. And mentor should be assigned to the course via the courses library

Reply:
still partly relevant:
1 course has has 10 cources
Example from real life - your c# course has inside some more courses - cooking course, photography course, UI/UX course.

Course should not have array of courses, courses library should have array of courses

Reply:
Done